src/app/admin/feed/_components/FeedImage.tsx

```
@@ -0,0 +1,38 @@
+type FeedImageProps = {
+  feed: Feed;
+  priority: boolean;
+  onClick: () => void;
+};
+
+export function FeedImage({ feed, priority, onClick }: FeedImageProps) {
+  const [loaded, setLoaded] = useState(false);
+
+  return (
+    <Flex
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)